components/Auth.js

```diff
@@ -52,21 +29,40 @@ export default function Auth() {
   }, [])
 
   return (
-    <View>
-      {user && <Text>Hi, {user.displayName}!</Text>}
-      <View style={styles.buttons}>
-        {user ? (
-          <Button title="Logout" onPress={() => firebase.auth().signOut()} />
-        ) : (
-          <Button title="Login" onPress={() => promptAsync()} />
-        )}
+    <View style={styles.container}>
+      <Title style={styles.title}>{UI_STRINGS.title}</Title>
+      <Subheading style={styles.subheading}>{UI_STRINGS.subheading}</Subheading>
+      <View>
+        <Button
+          style={styles.button}
+          accessibilityLabel="login"
+          mode="contained"
+          icon="google"
+          onPress={() => handleLogin()}
+        >
+          {UI_STRINGS.button}
+        </Button>
       </View>
     </View>
   )
 }
 
 const styles = StyleSheet.create({
-  buttons: {
-    marginTop: 8,
+  container: {
+    alignItems: 'center',
+    justifyContent: 'center',
+    display: 'flex',
+  },
+  button: {
+    textAlign: 'center',
+    alignItems: 'center',
+    padding: 8,
+  },
+  title: {
+    fontSize: theme.text.small,
+    marginBottom: theme.spacing.tiny,
+  },
+  subheading: {
+    marginBottom: theme.spacing.medium,
   },
```

Comment:
I would recommend that we use the typography components from paper for typography instead of spreading the codebase with font sizes.

for marging, but this is a personal preference, I would prefer a spacing component instead of using marging. the spacing component would work similar to the spacing unit in the theme, applying spacing in multipliers of the base unit. see the prototype I shared for a possible implementation of a spacer component